sqlbrite/src/main/java/com/squareup/sqlbrite/QueryToOneOperator.java

```diff
@@ -9,11 +9,11 @@
 
 final class QueryToOneOperator<T> implements Observable.Operator<T, SqlBrite.Query> {
   private final Func1<Cursor, T> mapper;
-  private final boolean emitNull;
```

Comment:
We can still emit `null` by passing it to the `mapToOneOrDefault` method (`null` passed as a second argument). Eventually, there could be two methods `mapToOneOrDefault` and `mapToOneOrNull` as it was previously.